book/src/validator-management.md

```diff
@@ -0,0 +1,183 @@
+```
+
+There is no `validator_definitions.yml` file present, so we can run `lighthouse
+vc` (**without** `--disable-auto-discover`) and it will create the following `validator_definitions.yml`:
+
+```yaml
+---
+- enabled: true
+  voting_public_key: "0xa5566f9ec3c6e1fdf362634ebec9ef7aceb0e460e5079714808388e5d48f4ae1e12897fed1bea951c17fa389d511e477"
+  type: local_keystore
+  voting_keystore_path: /home/paul/.lighthouse/validators/sally/one/voting-keystore.json
+  voting_keystore_password_path: /home/paul/.lighthouse/secrets/0xa5566f9ec3c6e1fdf362634ebec9ef7aceb0e460e5079714808388e5d48f4ae1e12897fed1bea951c17fa389d511e477
+- enabled: true
+  voting_public_key: "0xaa440c566fcf34dedf233baf56cf5fb05bb420d9663b4208272545608c27c13d5b08174518c758ecd814f158f2b4a337"
+  type: local_keystore
+  voting_keystore_path: /home/paul/.lighthouse/validators/sally/two/voting-keystore.json
+  voting_keystore_password_path: /home/paul/.lighthouse/secrets/0xaa440c566fcf34dedf233baf56cf5fb05bb420d9663b4208272545608c27c13d5b08174518c758ecd814f158f2b4a337
+- enabled: true
+  voting_public_key: "0x87a580d31d7bc69069b55f5a01995a610dd391a26dc9e36e81057a17211983a79266800ab8531f21f1083d7d84085007"
+  type: local_keystore
+  voting_keystore_path: /home/paul/.lighthouse/validators/john/voting-keystore.json
+  voting_keystore_password_path: /home/paul/.lighthouse/secrets/0x87a580d31d7bc69069b55f5a01995a610dd391a26dc9e36e81057a17211983a79266800ab8531f21f1083d7d84085007
+```
+
+All `voting-keystore.json` files have been detected and added to the file.
+Notably, the `sally/three/my-voting-keystore.json` file was *not* added to the
+file, since the file name is not exactly `voting-keystore.json`.
+
+In order for the validator client to decrypt the validators, they will need to
+ensure their `secrets-dir` is organised as below:
+
+```
+~/.lighthouse/secrets
+├── 0xa5566f9ec3c6e1fdf362634ebec9ef7aceb0e460e5079714808388e5d48f4ae1e12897fed1bea951c17fa389d511e477
+├── 0xaa440c566fcf34dedf233baf56cf5fb05bb420d9663b4208272545608c27c13d5b08174518c758ecd814f158f2b4a337
+└── 0x87a580d31d7bc69069b55f5a01995a610dd391a26dc9e36e81057a17211983a79266800ab8531f21f1083d7d84085007
+```
+
+
+### Manual configuration
+
+The automatic validator discovery process works out-of-the-box with validators
+that are created using the `lighthouse account validator new` command. The
+details of this process are only interesting to those who are using keystores
+generated with another tool or have a non-standard requirements.
+
+If you are one of these users, manually edit the `validator_definitions.yml`
+file to suit your requirements. If the file is poorly formatted, the validator
+client will refuse to start. However, if one or more of the validators have
+incorrect paths the validator client will still start and attempt to run
+whichever validators it can. As such, care should be taken to observe the logs
+when starting the validator client to ensure it has been able to initialize
+*all* validators.
```

Comment:
If it isn't too hard to implement: I think the user would expect in this case a hard (and loud) failure if any of the validators hasn't been initialized.

Reply:
Agreed, I wasn't quite sure about this one. I was originally on your side but I heard another client was still running even if some validators were bad.

I think we might as well stick to our guns here and fail fast and loud :)